pkg/vcs/git.go

```diff
@@ -114,7 +111,10 @@ func (git *git) CheckoutBranch(repo, branch string) (*Commit, error) {
 	if err != nil {
 		return nil, err
 	}
-	if _, err := git.git("checkout", "FETCH_HEAD"); err != nil {
+	if _, err := git.git("checkout", "FETCH_HEAD", "--force"); err != nil {
+		return nil, err
+	}
+	if err := git.repair(); err != nil {
```

Comment:
`repair()` also handles the case when the repository is irrecoverably damaged, so I think it's better to let it stay at the beginning of the function. Otherwise all commands above will just fail.

Reply:
Ah that makes sense, thanks. I added git.repair() back to the beginning of the function, but also kept it after git checkout(), because if a directory was present in the original branch but not the new checkout it wasn't being removed otherwise.